docs/introduction.md

```diff
@@ -0,0 +1,38 @@
+## Motivation
```

Comment:
Mention that there is a CLI tool for managing platforms, brokers and instances? It's in the picture but there are no details about it